gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=1.0.0-SNAPSHOT
+projectVersion=1.0.0.M2-SNAPSHOT
```

Comment:
the M suffix was automatically removed, so I guess we're working on M2 now? Or is that gonna be the first release?

Reply:
we don't typically include the M2 in the snapshot name in other projects